test/integration/ddl_sharding_key_test.lua

```diff
@@ -0,0 +1,370 @@
+end)
+
+pgroup:set_after_all(function(g) helpers.stop_cluster(g.cluster) end)
+
+pgroup:set_before_each(function(g)
+    helpers.truncate_space_on_cluster(g.cluster, 'customers')
+    helpers.truncate_space_on_cluster(g.cluster, 'writers')
+end)
+
+local function check_get(g, space_name, id, name)
+    local result, err = g.cluster.main_server.net_box:call('crud.get', {
+        space_name, {id, name}
+    })
+
+    t.assert_equals(err, nil)
+    t.assert(result ~= nil)
+end
+
+pgroup:add('test_insert_object_get', function(g)
+    -- insert_object
+    local result, err = g.cluster.main_server.net_box:call(
+        'crud.insert_object', {'customers', {id = 1, name = 'Fedor', age = 59}})
+
+    t.assert_equals(err, nil)
+    t.assert_equals(result.metadata, {
+        {is_nullable = false, name = 'id', type = 'unsigned'},
+        {is_nullable = false, name = 'bucket_id', type = 'unsigned'},
+        {is_nullable = false, name = 'name', type = 'string'},
+        {is_nullable = false, name = 'age', type = 'number'},
+    })
+    local objects = crud.unflatten_rows(result.rows, result.metadata)
+    t.assert_equals(objects, {{id = 1, bucket_id = 86, name = 'Fedor', age = 59}})
+
+    -- get
+    local result, err = g.cluster.main_server.net_box:call('crud.get', {'customers', {1, 'Fedor'}})
+
+    t.assert_equals(err, nil)
+    t.assert(result ~= nil)
+    local objects = crud.unflatten_rows(result.rows, result.metadata)
+    t.assert_equals(objects, {{age = 59, bucket_id = 86, id = 1, name = "Fedor"}})
+
+    -- insert_object again
+    local obj, err = g.cluster.main_server.net_box:call(
+        'crud.insert_object', {'customers', {id = 1, name = 'Alexander', age = 37}})
+
+    t.assert_equals(err, nil)
+    t.assert_equals(obj.rows, {{1, 1690, "Alexander", 37}})
+end)
+
+pgroup:add('test_insert_get', function(g)
+    -- insert
+    local result, err = g.cluster.main_server.net_box:call(
+        'crud.insert', {'customers', {2, box.NULL, 'Ivan', 20}})
+
+    t.assert_equals(err, nil)
+    t.assert_equals(result.metadata, {
+        {is_nullable = false, name = 'id', type = 'unsigned'},
+        {is_nullable = false, name = 'bucket_id', type = 'unsigned'},
+        {is_nullable = false, name = 'name', type = 'string'},
+        {is_nullable = false, name = 'age', type = 'number'},
+    })
+    t.assert_equals(result.rows, {{2, 1366, "Ivan", 20}})
+
+    -- get
+    local result, err = g.cluster.main_server.net_box:call('crud.get', {'customers', {2, 'Ivan'}})
+
+    t.assert_equals(err, nil)
+    t.assert(result ~= nil)
+    t.assert_equals(result.rows, {{2, 1366, "Ivan", 20}})
+end)
+
+pgroup:add('test_update', function(g)
+    local tuple = {2, box.NULL, 'Ivan', 20}
+
+    local update_operations = {
+        {'+', 'age', 10},
+    }
+
+    -- insert tuple
+    local result, err = g.cluster.main_server.net_box:call('crud.insert', {
+        'customers', tuple
+    })
+
+    t.assert_equals(err, nil)
+    t.assert(result ~= nil)
```

Comment:
Same as above. Please apply these changes to the entire code below.

Reply:
Fixed, thanks.